index.html

```diff
@@ -5,8 +5,36 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-    
+    <h1>About Erin Gurley</h1>
+    <p>My name is Erin and this is a little bit about me.</p>
+    <img src= "assets/pic.jpg" alt id=picture of girl>
+    <ul>
+        <li><a href="#Likes">Likes</a></li>
+        <li><a href="#Dislikes">Dislikes</a></li>
+    </ul>
+    <hr> 
+    <div id="Likes"> 
+        <h2>Likes</h2>
+    <ol>
+        <li>Writing and performing poetry.</li>
+        <li>Traveling to new places.</li>
+        <li>Riding motorbikes.</li>
+        <img src= "assets/vietnam.jpg" alt id=picture of vietnam>
+    </ol>
+    </div>
+    <hr>
+    <div id="Dislikes"> 
+            <h2>Dislikes</h2>
+        <ol>
+            <li>Mustard.</li>
+            <li>Walking up multiple flights of stairs.</li>
+            <li>Flossing.</li>
+            <img src= "assets/flossing.jpg" alt id=picture of flossing>
+        </ol>
+        </div>
+
```

Comment:
Include a footer tag with a link to your GitHub so your fans can find you :-) 
`<footer><a href="https://www.github.com/eringurley">by Erin Gurley</a></footer>`